### PR TITLE
- added styles for cards block

### PR DESCRIPTION
**Pull request by irenelagno**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix https://github.com/hlxsites/hsf-commonmoves/issues/8

Test URLs:
- Before: https://main--hsf-commonmoves--hlxsites.hlx.page/drafts/cards
- After: [https://cards-block--hsf-commonmoves--hlxsites.hlx.page/drafts/cards](https://cards-block--hsf-commonmoves--hlxsites.hlx.page/drafts/cards)
- repo https://adobe.sharepoint.com/:w:/r/sites/HelixProjects/_layouts/15/Doc.aspx?sourcedoc=%7B564955A9-FFFC-4BE0-A6AF-218836CC0451%7D&file=cards.docx&action=default&mobileredirect=true

### CODE REVIEW

File: styles/lazy-styles.css [hunk 1]
```diff
@@ -14,3 +14,10 @@
   font-weight: 400;
   font-style: normal;
 }
+
+@font-face {
```
Comment (bstopp) [inline styles/lazy-styles.css:18]: You shouldn't need this.

File: styles/styles.css [hunk 2]
```diff
@@ -17,16 +17,21 @@
   /* Colors */
   --body-color: #3a3a3a;
   --secondary-button-background: #fff;
+  --lightgray: #f6f6f6;
+  --black: #000;
 
   /* Fonts */
   --font-family-georgia: 'GeorgiaPro', 'GeorgiaPro Fallback', 'Arial', sans-serif;
   --font-family-proxima: 'ProximaNova', 'ProximaNova Fallback', 'Arial', sans-serif;
+  --font-family-proxima-semibold: 'ProximaSemibold', 'ProximaSemibold Fallback', 'Arial', sans-serif;
```
Comment (bstopp) [inline styles/styles.css:26]: Reference the Proxima and use 600 font-weight instead.

**Review by bstopp — CHANGES_REQUESTED** [review]

File: styles/lazy-styles.css [hunk 3]
```diff
@@ -14,3 +14,10 @@
...
+  font-family: ProximaSemibold;
+  src: url('./fonts/woff2/ProximaNova-Semibold.woff2') format('woff2'), url('./fonts/woff/ProximaNova-Semibold.woff') format('woff'), url('./fonts/ttf/ProximaNova-Semibold.ttf') format('truetype');
+  font-weight: 400;
+  font-style: normal;
+}
```
Comment (bstopp) [inline styles/lazy-styles.css:23]: This should be:

```
font-family: ProximaNova
font-weight: 600
```

**Review by bstopp — CHANGES_REQUESTED** [review]

File: blocks/cards/cards.css [hunk 4]
```diff
@@ -1,31 +1,88 @@
...
+.cards-container .default-content-wrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 32px 0;
+}
+
+.cards-container .default-content-wrapper h4 {
+  text-transform: uppercase;
+  font-family: var(--font-family-proxima-semibold);
```
Comment (bstopp) [inline blocks/cards/cards.css:65]: change this to family-proxima

add `font-weight: 600`

File: styles/lazy-styles.css (hunk 3 above)
Comment (irenelagno) [reply styles/lazy-styles.css:24]: updated

File: blocks/cards/cards.js [hunk 5]
```diff
@@ -1,18 +1,17 @@
...
+      if (div.querySelector('picture')) {
+        // update container for picture with label
+        div.className = 'cards-card-image';
+        if (div.lastChild.nodeType === Node.TEXT_NODE) {
+          const picture = div.querySelector('picture');
+          const paragraphElement = document.createElement(('p'));
+          paragraphElement.append(div.lastChild);
+          div.append(picture, paragraphElement);
+        }
+      } else div.className = 'cards-card-body';
```
Comment (bstopp) [inline blocks/cards/cards.js:14]: If i understand this correctly. The Label for the picture will be whatever text is on the left side of the card block, under the picture. Correct? 

Are we sure that there isn't already a paragraph tag made?

File: styles/floating-images.css [hunk 6]
```diff
@@ -103,7 +103,7 @@ main .section.image-left .default-content-wrapper .content .button-container {
     }
 }
 
-@media (max-width: 600px) {
+@media (max-width: 599px) {
```
Comment (bstopp) [inline styles/floating-images.css:106]: Why the change?

File: blocks/cards/cards.js (hunk 5 above)
Comment (irenelagno) [reply blocks/cards/cards.js:14]: This is a good call. [Here is an example](https://adobe.sharepoint.com/:w:/r/sites/HelixProjects/_layouts/15/Doc.aspx?sourcedoc=%7B564955A9-FFFC-4BE0-A6AF-218836CC0451%7D&file=cards.docx&action=default&mobileredirect=true) of how we expect the author to add a label. We can add as a note always add a label from the new line to avoid this transformation

File: styles/floating-images.css (hunk 6 above)
Comment (irenelagno) [reply styles/floating-images.css:106]: I did additional testing and find out that changes for floating image happens when we change dimensions from 599->600

Comment (bstopp) [reply styles/floating-images.css:106]: Is it going to break something if we settle on 600px for everything?

File: blocks/cards/cards.css [hunk 7]
```diff
@@ -1,31 +1,128 @@
...
+  .cards.block {
+    flex-direction: row;
+  }
+
+  .cards.three-column .cards-item {
+    width: 33.33%;
+    margin-bottom: 0;
+  }
+
+  .cards.two-column .cards-item {
```
Comment (bstopp) [inline blocks/cards/cards.css:117]: Rather than make the authors have to specify the number of columns. Can we programmatically set the class name based on how many cards are found in the block. This is [how the Columns block works](https://github.com/hlxsites/hsf-commonmoves/blob/fce57d1869411ccb919c78298486f03322fb9f7f/blocks/columns/columns.js#L3)

File: styles/floating-images.css (hunk 6 above)
Comment (irenelagno) [reply styles/floating-images.css:106]: nope. I think it should work fine

File: blocks/cards/cards.css (hunk 7 above)
Comment (irenelagno) [reply blocks/cards/cards.css]: yep, it works. Updated

File: styles/floating-images.css (hunk 6 above)
Comment (bstopp) [reply styles/floating-images.css:106]: Let's bump this to 600px and that should be last change for this.

**Review by bstopp — APPROVED** [review]
